DoSomething/quasar
# Pipenv

#### What's this PR do?
* Updates code to use [Pipenv[(https://docs.pipenv.org/en/latest/) instead of `requirements.txt` file for dependency tracking. I made sure all dependencies from `requirements.txt` were imported to Pipenv.
* Removes `requirements.txt`.
* Updates documentation with `pipenv` commands, removed Vagrant section for local database dev, and expands on linting rules.
#### Where should the reviewer start?
https://github.com/DoSomething/quasar/compare/pipenv?expand=1#diff-1e61a31bf9b94805f869dc4137ec1885
#### How should this be manually tested?
I manually ran a local install and build with `pipenv` with no issues. Before merge to master, I'll update the Jenkins QA `Deploy Branch` job to use `pipenv` commands and see if it works well.
